mifosng-provider/src/main/java/org/mifosplatform/portfolio/account/domain/AccountTransferRepository.java

```diff
@@ -14,10 +14,10 @@
 
 public interface AccountTransferRepository extends JpaRepository<AccountTransferTransaction, Long>, JpaSpecificationExecutor<AccountTransferTransaction> {
 
-    @Query("from AccountTransferTransaction att where att.accountTransferDetails.fromLoanAccount.id= :accountNumber and att.reversed=false")
+    @Query("from AccountTransferTransaction att where att.accountTransferDetails.fromLoanAccount.id= :accountNumber and att.reversed=false and att.accountTransferDetails.transferType !=5  order by att.id desc")
```

Comment:
Should be replaced with a new method which accepts loan transaction ids as input to find reversible account transfers (in this case it is refund transactions) anyway I made changes will send pull request for same